components/HackathonModal.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState, useEffect } from 'react';
+import Modal from 'react-modal';
+import Image from '../components/Image';
+import Link from 'next/link';
+import styles from '../styles/components/HackathonModal.module.scss';
+import { website_config } from '../config';
+
+const MODAL_DISPLAY_INTERVAL = 2 * 24 * 60 * 60 * 1000; // 2 days in milliseconds
```

Comment:
I think moment offers resources for this that make it more readable. https://momentjs.com/docs/#/durations/ something like `moment.duration(2, 'days').milliseconds()`